tests/conftest.py

```diff
@@ -147,3 +150,49 @@ def cmdline_test(request):
     # Import qutebrowser.app so all cmdutils.register decorators get run.
     import qutebrowser.app  # pylint: disable=unused-variable
     return request.param
+
+
+class ConfigStub:
+
```

Comment:
That's intentional - I'm not sure if [pep257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) (the document) says anything about it, but [pep257](https://pypi.python.org/pypi/pep257/) (the tool) does it this way:

`D20{3,4}	1 blank required {before,after} class docstring.`